src/core/libmaven/Compound.h

```diff
@@ -12,29 +12,77 @@ class Fragment;
 using namespace std;
 
 class Compound{
-    /*
-    *@brief  -   class to represent a compound
-    *@detail  -  class Compound representing a compound. It will hold variable to
-    *discribe a compound and also PeakGroup object it will belong to in addition
-    *to other required functions for other operation
-    */
 
+    /**
+     *@brief  -   class to represent a compound
+     *@detail  -  class Compound representing a compound. It will hold variable to
+     *discribe a compound and also PeakGroup object it will belong to in addition
+     *to other required functions for other operation
+     */
     private:
-        /** 
-        *@param - One group represent one compund and
-        * _group will present a link to that compund
-         */
-        PeakGroup _group;
+
         /**
-        *@param - _groupUnlinked will check  wether Compound and PeakGroup are linked or not
+         *@param - compund id
         */
-        bool      _groupUnlinked;
+        string _id;
 
         /**
-         * Formula of the compound.
-         */
+         *@param -  name of compound
+        */
+        string _name;
+
+        /**
+         *@param -  formula of compound
+        */
         string _formula;
 
+        /**
+         *@param   -  alias name of compound
+        */
+        string _alias;
+
+        /**
+        *@param -  srmId will hold filterLine string from mzxml file which represent type of
+        *mass spec and ionization is used and other info as well
+        */
+        string _srmId;
+
+        float _expectedRt;
+
+        /**
+         *@param  -   number of charge of compound (electron loss or gain)
+        */
+        int _charge;
+
+        /**
+         *@param   -  mass of this compund
+        */
+        float _mass;
```

Comment:
In almost all cases, this member stores the compound's m/z (mass divided by charge) and not the mass. Only when the compound is initialized and charge is not available, is this set to the neutral mass. So, I suggest we rename it to `_mz` and change its corresponding setter and getter to `setMz()` and `mz()`, respectively.